taipy/core/_orchestrator/_orchestrator.py

```diff
@@ -38,6 +39,7 @@ class _Orchestrator(_AbstractOrchestrator):
     blocked_jobs: List = []
     lock = Lock()
     __logger = _TaipyLogger._get_logger()
+    _submission_entities: Dict[str, Submission] = {}
```

Comment:
Please refer to my comment here https://github.com/Avaiga/taipy/pull/655/files#r1444309709 for why this attributes were created